index.bs

```diff
@@ -1784,6 +1788,57 @@ endpoint is verified to be a WebTransport endpoint; thus, the Web application
 cannot distinguish between a non-existing endpoint and the endpoint that is not
 willing to accept connections from the Web.
 
+## Authentication using Certificate Hashes {#certificate-hashes}
+
+In the vast majority of cases, a user agent authenticates a TLS connection
+between itself and a remote endpoint by verifying the validity of the TLS
+server certificate provided against the server name in the URL; this approach
+will be further referred to as Web PKI.  The full details of how the Web PKI
+functions are implementation-defined, and go far beyond the scope of this
+docuement; in practice, those tend to be fairly complex, and involve
+procedures for maintaining roots of trust, handling revocations of previously
+issued certificates, as well as defending against key compromise, certificate
+mis-issuance, and numerous other security risks.
```

Comment:
I don't think that you need to elaborate on why Web PKI is big, important, and complex.  The choice of words here is not sufficiently neutral: "far beyond", "fairly complex", ...

You don't have to draw a comparison. You only have to define the fingerprint-based scheme in terms of how it works.  To that end, most of this paragraph can go.

Reply:
Deleted that part.